app/models/zitting.js

```diff
@@ -1,20 +1,23 @@
 import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
-import { notEmpty } from '@ember/object/computed';
+import { isEmpty } from '@ember/utils';
 
-export default Model.extend({
-  geplandeStart: attr('datetime'),
-  gestartOpTijdstip: attr('datetime'),
-  geeindigdOpTijdstip: attr('datetime'),
-  opLocatie: attr(),
-  bestuursorgaan: belongsTo('bestuursorgaan', { inverse: null }),
-  afgeleidUit: attr(),
-  behandeldeAgendapunten: hasMany('behandeling-van-agendapunt', { inverse: null }),
-  notulen: belongsTo('editor-document', { inverse: null }),
-  publicatieAgendas: hasMany('agenda', { inverse: null }),
-  agendapunten: hasMany('agendapunt', { inverse: 'zitting' }),
-  secretaris: belongsTo('functionaris', { inverse: null }),
-  voorzitter:  belongsTo('mandataris', { inverse: null }),
-  aanwezigenBijStart: hasMany('mandataris', { inverse: null }),
-  afwezigenBijStart: hasMany('mandataris', { inverse: null }),
-  isPublished: notEmpty('afgeleidUit')
-});
+export default class ZittingModel extends Model {
+  @attr('datetime') geplandeStart;
+  @attr('datetime') gestartOpTijdstip;
+  @attr('datetime') geeindigdOpTijdstip;
+  @attr opLocatie;
+  @attr afgeleidUit;
+  @belongsTo('bestuursorgaan', { inverse: null }) bestuursorgaan;
+  @hasMany('behandeling-van-agendapunt', { inverse: null }) behandeldeAgendapunten;
+  @belongsTo('editor-document', { inverse: null }) notulen;
+  @hasMany('agenda', { inverse: null }) publicatieAgendas;
+  @hasMany('agendapunt', { inverse: 'zitting' }) agendapunten;
+  @belongsTo('functionaris', { inverse: null }) secretaris;
+  @belongsTo('mandataris', { inverse: null }) voorzitter;
+  @hasMany('mandataris', { inverse: null }) aanwezigenBijStart;
+  @hasMany('mandataris', { inverse: null }) afwezigenBijStart;
+
+  get isPublished() {
```

Comment:
I don't think this is used anywhere?

Reply:
I think this is related to old code indeed, should be safe to be removed.